extension/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "manifest_version": 3,
+  "name": "Preact Signals DevTools",
+  "version": "1.0.0",
+  "description": "Debug and inspect Preact Signals in your web applications",
+  "permissions": ["activeTab", "scripting"],
+  "host_permissions": ["<all_urls>"],
+  "content_scripts": [
+    {
+      "matches": ["<all_urls>"],
+      "js": ["content.js"],
+      "run_at": "document_start",
+      "all_frames": true
+    }
+  ],
+  "devtools_page": "devtools.html",
+  "background": {
+    "service_worker": "background.js"
```

Comment:
doesn't make much difference but FYI these can be ES modules now:

```ts
  "background": {
    "service_worker": "background.js",
    "type": "module"
  },
```

you don't use anything module-specific but may still be good to know :D